test2code/build.gradle.kts

```diff
@@ -69,6 +69,9 @@ tasks {
     withType<KotlinCompile> {
         kotlinOptions.jvmTarget = "1.8"
     }
+    jar {
+        duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
reason for adding this

Reply:
If I delete it, Gradle will throw an error about being unable to override the generated sources, so I added it to fix the issue.

Reply:
Please double check, sources should not be overwritten during the fine configured build.